lua/nordic/colors/nordic.lua

```diff
@@ -1,6 +1,7 @@
 -- The Nord palette: https://www.nordtheme.com/.
 -- This file has a bunch of added colors.
 
+-- note: all hex codes must be uppercase (for testing)
```

Comment:
Now I am really nitpicking, but making this "NOTE: All..." is a bit more conventional and is the expected default for stuff like `todo-comments.nvim`.

Reply:
I see you use "note: ..." Quite a few times.  If you do not know about this in neovim, try:

`:% s/note:/NOTE:/g` :)

Reply:
On the note of the uppercase, maybe use `string.upper` in our `get_highlight` function?

Reply:
> Now I am really nitpicking, but making this "NOTE: All..." is a bit more conventional and is the expected default for stuff like `todo-comments.nvim`.

see: [fix(comments): replace note with NOTE;](https://github.com/AlexvZyl/nordic.nvim/pull/150/commits/7f6fec7577c982339aaccaa19e8219b2687a6f12)



> I see you use "note: ..." Quite a few times. If you do not know about this in neovim, try:
> 
> `:% s/note:/NOTE:/g` :)

Honestly, I have been using Helix for the past few months, but that is useful.


> On the note of the uppercase, maybe use `string.upper` in our `get_highlight` function?

`get_highlight` does always return uppercase which means `assert_eq` will fail if the color is defined using lowercase.